kubernetes/rancher/lib/docker-hello-world/playbooks/create.yaml

```diff
@@ -0,0 +1,14 @@
+---
+- hosts: all
+  vars:
+    ansible_python_interpreter: /usr/bin/python3
+  tasks:
+    - name: Run hello-world Docker container and join it to the Rancher network
+      docker_container:
+        name: hello-world
+        image: hello-world:linux
+        pull: true
+        state: started
+        labels:
+          io.rancher.container.network: "true"
```

Comment:
I don't think this does what you want it to do, does it? Running things in kubernetes means also configuring pods, services, ingresses and other seemingly unnecessary things, and this does none of it. I'd expect you to use a kubectl command to launch a service or something here, definitely not a Docker command.

Reply:
This was meant for Rancher to detect a running Docker container but I agree that the example would be better with some  examples using `kubectl` and `helm`. Do you have some suggestions on what to choose for the deployment?

Reply:
I think the standard thing would be to deploy a prometheus heml chart^1 using an ansible helm module^2. This should be fairly simple, since the whole point of a helm chart is to be simple.

1. https://artifacthub.io/packages/helm/prometheus-community/prometheus
2. https://docs.ansible.com/ansible/latest/collections/community/kubernetes/helm_module.html

You will need to install the community.kubernetes collection I think, since that's not included by default.

Reply:
Well this could be problematic as we talked. I'm able to deploy Rancher in a Docker container but for deploying Prometheus helm chart you would need a kubectl cluster config file which I wasn't able to retrieve from the Rancher container and I also believe that this is not the best practice. However, if I go to Rancher dashboard and copy kubectl config for the default `local` cluster everything works and Prometheus chart gets deployed.

Do you have some other ideas about that?

Reply:
I was able to overcome that issue by running Prometheus helm chart in a Rancher docker container with `docker exec` and everything seems to work fine.